### DISCUSSION TITLE
Recreational rockfish genus allocation to species posted

### DISCUSSION THREAD

**Discussion by JuliaCoates** [pfmc-assessments/california-data#8] [General]
Hi all.  I posted a table of recreational total mortality for the rockfish genus category allocated to species during the Covid period here: [5b3ff05](https://github.com/pfmc-assessments/california-data/commit/5b3ff059dce40d2e29958b460c52c1651be48a95).  Original and allocated values are shown for numbers of fish and weight.  A read me file gives more details and the script is also posted.  
@okenk 
@brianlangseth-NOAA 
@EJDick-NOAA 
@melissamonk-NOAA 
@chantelwetzel-noaa

**Comment by brianlangseth-NOAA** [comment]
@JuliaCoates Thanks for posting these. I notice the README talks about corrections for 2020 and 2021 but the tables for each species only show 2020 proxy values. I plan to add the proxy value to the RecFIN estimate in 2020 only. Please let me know if further adjustments for 2021 are also needed.

**Reply by chantelwetzel-noaa** [reply]
@brianlangseth-NOAA Julia is referring to the file called genus_allocate.csv from the linked commit above. This csv file contains values for both 2020 and 2021 where she allocated rockfish genus catch to specific species for these years.  There is also the proxy catch file, which I think you are referring to, that is for only 2020. Those values should be added to any existing catches for that year in RecFIN.

**Reply by brianlangseth-NOAA** [reply]
Thanks @chantelwetzel-noaa for pointing out my confusion. I got it now.

**Comment by chantelwetzel-noaa** [comment]
@JuliaCoates In the column called `orig_allocated` with two indicators: original or allocated can you clarify if we should only add quantities that are marked as allocated or should the sum of the original and allocated be added to the existing catches?  Thank you!

**Reply by JuliaCoates** [reply]
Yes, please use only the values marked allocated.  The original values are just posted for your reference and should match the values you currently have in your unallocated data.

**Reply by chantelwetzel-noaa** [reply]
@JuliaCoates Thank you!

**Comment by brianlangseth-NOAA** [comment]
During the process of confirming catch values for canary, the state clarified that the 2020 (`with orig_allocation == "allocated"`) values (which are for both PC and PR) should replace RecFIN estimates for that year. The 2021 allocated provided by @JuliaCoates values are only for PC mode, so those should be added to PR estimates from RecFIN to produce final estimates for that year.

**Reply by brianlangseth-NOAA** [reply]
Updated guidance we got in June of 2023 was to add `orig_allocated == "allocated"` values to estimates within RecFIN, not as a replacement.
